etc/institutions-auth.xsl

```diff
@@ -57,6 +57,19 @@
                             <suffix/>
                         </user>
                     </xsl:when>
+                    <!-- Brown University (BROWN) -->
+                    <xsl:when test="$idp='https://sso.brown.edu/idp/shibboleth'">
+                        <id>brown</id>
+                        <user>
+                            <username><xsl:value-of select="//attribute[@name='mail']/@value"/></username>
+                            <fullname><xsl:value-of select="//attribute[@name='displayName']/@value"/></fullname>
+                            <familyName><xsl:value-of select="//attribute[@name='sn']/@value"/></familyName>
+                            <givenName><xsl:value-of select="//attribute[@name='givenName']/@value"/></givenName>
+                            <isMemberOf><xsl:value-of select="//attribute[@name='isMemberOf']/@value"/></isMemberOf>
```

Comment:
As for **CAS** server setting changes, add the following line  to the `brown` block in the `institutions-auth.xsl ` file.

```
<isMemberOf><xsl:value-of select="//attribute[@name='isMemberOf']/@value"/></isMemberOf>
```

As for **Shibboleth** server setting changes, simply add the following line to the file `attribute-map.xml`.

```
<Attribute name="urn:oid:1.3.6.1.4.1.5923.1.5.1.1" id="isMemberOf"/>
```